docs/source/cpp/streaming_execution.rst

```diff
@@ -35,20 +34,23 @@ For many complex computations, successive direct :ref:`invocation of
 compute functions <invoking-compute-functions>` is not feasible
 in either memory or computation time. Doing so causes all intermediate
 data to be fully materialized. To facilitate arbitrarily large inputs
-and more efficient resource usage, Arrow also provides a streaming query
-engine with which computations can be formulated and executed.
+and more efficient resource usage, the Arrow C++ implementation also
+provides Acero (pronounced AH-serr-oh), a streaming query engine with which computations can
```

Comment:
```suggestion
provides Acero, a streaming query engine with which computations can
```

I think we only need the pronunciation in the overview, but if we do want it here we should copy what's up there too

Reply:
Let's keep it in the one spot so we have a single source of truth.  Good catch.